src/Evolution/Systems/Cce/OptionTags.hpp

```diff
@@ -319,6 +319,31 @@ struct EndTime : db::SimpleTag {
   }
 };
 
+/// \brief Represents the final time of a CCE evolution that should just proceed
+/// until it receives no more boundary data and becomes quiescent.
+struct NoEndTime : Tags::EndTime, db::SimpleTag {
```

Comment:
The default for the option tag is negative infinity, so this allows the behavior that the end time is taken from the h5 file if not specified in the input file options, otherwise is the value given in the input options. I'll modify the documentation for the derived tag to emphasize that behavior.

Reply:
but that should be handled by the SpecifiedEndTime case shouldn't it?  or are you allowing to specify a end time while reading from file that is not the last time in the file?

Reply:
Yes, I'm allowing the option of only extracting some portion of a worldtube file. Most production runs will want to run over the full file, but I've found it useful for testing and diagnostic to do short runs over segments of the inspiral.

Reply:
okay, then modifying the documentation is fine